### PR TITLE
docs(connectors): added guide to run custom connectors in helm

### PR DESCRIPTION
**Pull request by igpetrov**
## Description

docs(connectors): added guide to run custom connectors in helm

## When should this change go live?

Any time

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by christinaausley** [comment]
@igpetrov always on it with the assignee and appropriate labels 👏

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+The default runtime loads Connectors from classpath via SPI. For the custom Connectors, there is a dedicated folder
+inside a **Connectors** Docker image `/opt/custom`. Any jar that is placed there will be included into the classpath.
+
+This page explains how to put your custom Connector into the `/opt/custom`.
+
+## Prerequisites
+
+Start with [creating and building](/components/connectors/custom-built-connectors/connector-sdk/) a 'fat' jar (jar with dependencies) of your custom **Connector**. For the purpose of
+this guide, let's consider this guide, let's consider the custom **Connector** name as `custom-connector-0.0.1-with-dependencies.jar`.
```
Comment (sbuettner) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md:19]:
```suggestion
this guide, let's consider the custom **Connector** name as `custom-connector-0.0.1-with-dependencies.jar`.
```

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Then, you need to place the 'fat' jar somewhere accessible by Helm during installation. For the purpose of this guide,
+let's consider, the path to the **Connector** is `https://my.host:80/dist/custom-connector-0.0.1-with-dependencies.jar`.
```
Comment (sbuettner) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md:22]:
```suggestion
let's consider the path to the **Connector** is `https://my.host:80/dist/custom-connector-0.0.1-with-dependencies.jar`.
```

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Modify Connectors config
+
+Modify `connectors` properties. The most straightforward way to do so is to modify `values.yaml` in the
+[Camunda Helm Charts repository](https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/values.yaml).
```
Comment (sbuettner) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md:27]:
```suggestion
Modify the `values.yaml` file in the [Camunda Helm Charts repository](https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/values.yaml).
```

**Review by sbuettner — APPROVED** [review]

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md (hunk 2 above)
Comment (igpetrov) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md:22]: Done

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md (hunk 3 above)
Comment (igpetrov) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md:27]: Done (in next rev)

**Review by sbuettner — DISMISSED** [review]

**Comment by igpetrov** [comment]
Ready for merge

**Review by christinaausley — APPROVED** [review]

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+---
+id: running-custom-connectors
+title: "Running custom Connectors"
+description: "Run custom Connectors in your Helm Kubernetes cluster."
+---
+
+You can deploy your custom **Connector** in your Helm Kubernetes cluster along with Connectors Bundle.
```
Comment (christinaausley) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md:7]: I need to add Connectors Bundle to the glossary to determine casing, an FYI to myself 👍

**Comment by igpetrov** [comment]
@christinaausley would it be okay if I merge it?

**Comment by christinaausley** [comment]
Done @igpetrov 👍

**Pull request merged**